src/components/SliderBox/mocks/slidesData.ts

```diff
@@ -0,0 +1,22 @@
+interface SlidesType {
+  name: string;
+  date: string;
+}
```

Comment:
move to the types folder. And you can use just `Slides` for naming

Reply:
fixed